server/service/src/sync/sync_status/status.rs

```diff
@@ -113,20 +129,20 @@ pub fn get_latest_sync_status(
         pull_central: pull_central_start_datetime.map(|started| SyncStatusWithProgress {
             started,
             finished: pull_central_done_datetime,
-            total_progress: pull_central_progress_start.map(i32_to_u32).unwrap_or(0),
-            done_progress: pull_central_progress_done.unwrap_or(0) as u32,
+            total_progress: pull_central_progress_total.map(i32_to_u32),
```

Comment:
:nit `total_progress`? isn't this just `total` 
I don't see what the progress adds to the name

Reply:
291d72aa0f3f96df67c9a944ad6b066c42ffe148